### PR TITLE
[Merged by Bors] - Update to operator-rs 0.27.1

### PR DESCRIPTION
**Pull request by maltesander**
# Description

fixes https://github.com/stackabletech/hbase-operator/issues/270

test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hbase-operator-it-custom/62/

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: deploy/manifests/crds.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+---
```
Comment (maltesander) [inline deploy/manifests/crds.yaml:2]: Not sure where this is coming from, needs some investigation (only did `make regenerate-charts`)

Comment (maltesander) [reply deploy/manifests/crds.yaml:2]: @siegfriedweber says this is expected behavior.

**Review by razvan — APPROVED** [review]
lgtm. one comment is obsolete but I cannot delete it anymore.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,10 @@
 ### Changed
 
 - Updated stackable image versions ([#275]).
+- `operator-rs` `0.2402` -> `0.27.1` ([#277]).
```
[the remark this answers is not in the document]
Comment (maltesander) [reply CHANGELOG.md]: done https://github.com/stackabletech/hbase-operator/pull/277/commits/4eea6c4451a80dc71e2767bd84ed40c734e1abb5

File: rust/crd/src/lib.rs [hunk 3]
```diff
@@ -125,10 +130,32 @@ impl HbaseRole {
             ],
         }
     }
+
+    pub fn roles() -> Vec<String> {
```
[the remark this answers is not in the document]
Comment (maltesander) [reply rust/crd/src/lib.rs]: good catch, i think it was for an error i did not end up using https://github.com/stackabletech/hbase-operator/pull/277/commits/f7224e4d4e789f12a3536b7777dc2c573cfb2826

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**